src/viam/sdk/components/camera/camera.cpp

```diff
@@ -37,7 +37,7 @@ std::shared_ptr<ResourceRegistration> Camera::resource_registration() {
     const google::protobuf::ServiceDescriptor* sd =
         p->FindServiceByName(viam::component::camera::v1::CameraService::service_full_name());
     if (!sd) {
-        throw std::runtime_error("Unable to get service descriptor for the camera service");
+        throw Exception("Unable to get service descriptor for the camera service");
```

Comment:
This Exception (and analogous in other resources) will disappear on rebase (fyi).